### PR TITLE
smalltalkCI and Pharo support

### PR DESCRIPTION
**Pull request by jchromik**
Fabio renamed filetreeCI smalltalkCI and added Pharo support. We updated the documentation accordingly.

### CODE REVIEW

**Comment by fniephaus** [comment]
\*friendly ping\*

**Comment by joshk** [comment]
Sorry for the delay, our wonderful @BanzaiMan is on holiday this week, we will look into this first thing next week.

One thing I do notice is the `BASELINE` env var, maybe this should be a setting in the .travis.yml? eg. `baseline: MyProjectBaseline` (maybe the key `baseline` is a little ambiguous)

**Comment by jchromik** [comment]
Done (see [#546](https://github.com/travis-ci/travis-build/pull/546) in [travis-ci/travis-build](https://github.com/travis-ci/travis-build))! Changes in the docs are in progress right now.

**Comment by fniephaus** [comment]
Nice! And good idea, @joshk. Thanks

**Comment by fniephaus** [comment]
Sorry for pinging once more, but I'm actually waiting for this to go live, so I can send out an announcement to our mailing lists.

File: user/languages/smalltalk.md [hunk 1]
```diff
@@ -25,32 +24,30 @@ and cc [@bahnfahren](https://github.com/bahnfahren),
...
+baseline: MyProjectBaseline
 ```
 
 ## Configuration options
 
-By default Travis CI will use `Squeak5.0` release. It is also possible
-to test projects against `SqueakTrunk`, `Squeak4.6` and `Squeak4.5`. To do so, set the
-`smalltalk` key in `.travis.yml`. For example, to test against both `Squeak5.0` and
-the `SqueakTrunk`:
+Next to the default image it is also possible to test projects against other versions of Squeak/Smalltalk or Pharo (an exhaustive list of supported images can be found at [smalltalkCI's GitHub repository](https://github.com/hpi-swa/smalltalkCI#images)). Also you can test multiple baselines at once.
```
Comment (plaindocs) [inline user/languages/smalltalk.md]: Could you reword to something like

"As well as using the default image you can also test test projects against other versions of Squeak/Smalltalk or Pharo (an full list of supported images can be found at [smalltalkCI's GitHub repository](https://github.com/hpi-swa/smalltalkCI#images))."

**Comment by plaindocs** [comment]
Otherwise I'm happy to merge this in (and maybe fine tune slightly after).

**Comment by jchromik** [comment]
Thanks for the feedback. We changed the docs as requested.  
Let us know if there are any further issues.

**Comment by fniephaus** [comment]
We can this be merged?

**Comment by jchromik** [comment]
Since smalltalkCI evolved since last November we made some updates.
@plaindocs could you please have a look and give us feedback?

If there are no issues we would be happy if this could be merged.

**Comment by fniephaus** [comment]
Maybe open a new PR for discussing this?

**Comment by plaindocs** [comment]
hi - sorry I missed the recent batch of notifications. Will take a look asap.

**Pull request closed without merging**